src/domain/task-system/services/src/task_executor_impl.rs

```diff
@@ -21,32 +22,29 @@ use kamu_core::{
 };
 use kamu_task_system::*;
 
-/////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
for some strange reason, we sometimes have these lines that are 89 characters long